lnwallet/transactions_test.go

```diff
@@ -649,218 +446,189 @@ var testCases = []testCase{
 func TestCommitmentAndHTLCTransactions(t *testing.T) {
 	t.Parallel()
 
-	tc, err := newTestContext()
-	if err != nil {
-		t.Fatal(err)
+	for _, test := range testCases {
+		t.Run(test.name, func(t *testing.T) {
+			testVectors(t, test) // nolint:scopelint
```

Comment:
bind `test := test` to avoid nolint?

Reply:
Fixed. But I was wondering why it complains? `test` isn't a reference

Reply:
indeed, i don't think there is actually a problem. i would attribute it to either the linter being overly cautious (not distinguishing between ptr and value) or just a bug